docs/readmes/lte/sessiond.md

```diff
@@ -80,3 +80,72 @@ On SessionD restart, session state will be synced to MME service, which
+will send a report to OCS or PCRF with used units
+4. PCRF/OCS will then send back another grant, which will be added to the still
+remaining grant. 
+5. PCRF/OCS can also include an indication to let sessiond know that there is 
+no more quota available for that user. So sessiond will only have a small 
+percentage left (around 20%) until is exhausted and execute an specific
+action
+
+See that PCRF or OCS can request to track the credit using Tx only, Rx only, Tx and Rx,
+TX, RX and Total. The tracking type will be determined by the very first grant received
+and will remain the same for the entire session for that specific monitor key (Gx reporting)
+or charging key (Gy reporting)
+
+To better understand the exhaustion algorithm refer to `SessionCredit::is_quota_exhausted`
+
+Below you have some differences on how sessiond deals with GX and Gy reporting 
+
+### PCRF (Gx reporting) 
+Gx reporting will not cause a session termination. Gx reporting will report credit
+back to PCRF until quota is exhausted. Once quota is exhausted, the monitor will be
+deleted from sessiond (stop reporting) but the session or the policy related with 
+that monitoring key will not be removed.
+
+Monitor removal will be triggered by PCRF sending a grant with one of its components
+(tx, rx or total) set to 0 (or not set). Note that this component (tx, rx or total) 
+must be one of the tracked types. So if we are tracking TX and RX, the deletion of
+the monitor will be triggered if we receive either a 0 on Tx or Rx 
+(total component will be ignored).
+
+When sessiond receives a 0 on one of its tracked components of the last received
+grant, the monitor will not be deleted immediately. Sessiond will exhaust the remaining 
+credit still left (remember we still have around 20% left of the previous grant), and 
+then stop reporting removing the monitoring key.
+
+### OCS (Gy reporting) 
+Gy reporting will cause session termination, redirection or restriction. Gy reporting will 
+report credit back to OCS until quota is exhausted. Then, and depending on 
+Final Unit Action, the session may be terminated, redirected or restricted.
+In all cases, this will have impact on the whole session, not on a specific policy of 
+that session
+
+Session action will be triggered by OCS sending a grant which must include AVP 
+Final Action Indication. If this AVP is not included, sessiond will keep tracking
+and reporting to OCS, even if quota is exhausted, but it will not take any action. 
+The reason for that is that we don't want to trigger an early termination of the 
+session due to OCS being too slow to provide a grant. Also, Per 3gpp 
+Final Unit Indication AVP is needed to trigger any action by sessiond.
+
+Note when sessiond receives a grant which includes Final Unit Indicaton, it 
+will continue tracking this charging key until any of its quotas is exhausted. 
+Once it is exhausted, Final Unit Action (termnate, redirect, restrict) 
+will be executed.
```

Comment:
Might be good to mention that the FUA will be lifted when an additional quota is given for the credit.